docs/pages/en-us/Client-API.md

```diff
@@ -26,9 +26,10 @@ The order of instantiation and use of these objects for sending audit events is
 
 #### Direct to Elasticsearch Configuration  
 Configuration required to be supplied via environment variable:  
-- `CAF_ELASTIC_HOST_AND_PORT_VALUES`: A comma separated list of hostnames and ports to use when contacting elasticsearch. eg. localhost:9200,otherHost:9200  
 - `CAF_ELASTIC_NUMBER_OF_SHARDS`: The number of shards elasticsearch is configured to use.  
-- `CAF_ELASTIC_NUMBER_OF_REPLICAS`: The number of replicas configured for elasticsearch.  
+- `CAF_ELASTIC_NUMBER_OF_REPLICAS`: The number of replicas configured for elasticsearch.
+- 'CAF_ELASTIC_HOST_VALUES': A comma separated list of hostnames to use when contacting elasticsearch. eg. localhost, otherHost
+- 'CAF_ELASTIC_PORT_VALUE': The REST port of the ElasticSearch server listens on. e.g. 9200
```

Comment:
These config options don't look like the others.

Reply:
Comment ignored.

Reply:
They still don't look like the others.  Do you ever actually look at the readme files you're updating:
https://github.com/CAFAudit/audit-service/blob/132f15f8f2c150e046857592859491b8e147a958/docs/pages/en-us/Client-API.md#direct-to-elasticsearch-configuration